com.abagames.noiz2sa.json

```diff
@@ -0,0 +1,82 @@
+{
+  "app-id": "com.abagames.noiz2sa",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "noiz2sa",
+  "rename-desktop-file": "noiz2sa.desktop",
+  "rename-icon": "noiz2sa",
+  "rename-appdata-file": "noiz2sa.appdata.xml",
+  "finish-args": [
+    "--device=dri",
+    "--share=ipc",
+    "--socket=pulseaudio",
+    "--socket=wayland",
+    "--socket=x11"
+  ],
+  "modules": [
+    "shared-modules/glu/glu-9.0.0.json",
+    "shared-modules/SDL/SDL-1.2.15.json",
+    "shared-modules/SDL/SDL_mixer-1.2.12.json",
+    "bulletml-flatpak/com.abagames.bulletml.json",
+    {
+      "name": "noiz2sa",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://master.dl.sourceforge.net/project/noiz2sa/noiz2sa%20source_binary/Noiz2sa%200.51a/noiz2sa-0.51a.tar.gz",
+          "sha256": "a7e2b689a320b49cd87a5666b38b934528c65d5a212efca81a21fdd45c9f88e6"
+        },
+        {
+          "type": "patch",
+          "path": "patches/0001-Use-shared-assets-location-from-a-config-header.patch"
+        },
+        {
+          "type": "patch",
+          "path": "patches/0002-Remove-FoeCommand-class-namespace-from-method-in-cla.patch"
+        },
+        {
+          "type": "patch",
+          "path": "patches/0003-Fix-makefile.patch"
+        },
+        {
+          "type": "file",
+          "path": "noiz2sa.desktop"
+        },
+        {
+          "type": "shell",
+          "commands": [
+            "install -Dm644 noiz2sa.desktop /app/share/applications/noiz2sa.desktop"
```

Comment:
Put these in a `build-commands` not in `sources`.

Reply:
I did this on purpose because it feels easier to maintain when we have the script that deals with an asset closer to that asset. Is there a problem in keeping it like this?